app/presenters/two_factor_authentication/set_up_phone_selection_presenter.rb

```diff
@@ -1,15 +1,15 @@
 module TwoFactorAuthentication
-  class PhoneSelectionPresenter < SelectionPresenter
+  class SetUpPhoneSelectionPresenter < SetUpSelectionPresenter
+    def initialize(user:)
+      @user = user
+    end
+
```

Comment:
This constructor looks to be identical to the base logic in `SetUpSelectionPresenter`, so we could remove it?

```suggestion
```